rasa_nlu/train.py

```diff
@@ -62,6 +62,21 @@ def init():     # pragma: no cover
     return config
 
 
+def do_train_in_worker(config):
```

Comment:
Instead of copying the `do_train` method, can we instead call it and throw away trainer and interpreter?